### PR TITLE
changed encoding

### PR DESCRIPTION
**Pull request by nrecoder**
Updated encoding to read text file in utf16 encoding

### CODE REVIEW

File: sleepylyze/ioeeg.py [hunk 1]
```diff
@@ -137,12 +137,12 @@ def get_info(self):
         print("Patient identifier:", self.metadata['in_num'])
 
         # read the first line to check encoding
-        with open(self.metadata['filepath'], 'r') as f:
+        with open(self.metadata['filepath'], encoding='utf16') as f:
             line = f.readline()
         if line[3] == '\x00':
             f_encoding = 'utf-16-le'
         else:
```
Comment (jag2037) [inline sleepylyze/ioeeg.py:144]:
```suggestion
        # try opening the file without specifying encoding
        try:
            with open(self.metadata['filepath'], 'r') as f:
                line = f.readline()
            if line[3] == '\x00':
                f_encoding = 'utf-16-le'
            else:
                f_encoding = 'ascii'
        # if we can't open the file without specifying the encoding, assume it's utf16
        except UnicodeDecodeError:
```

**Review by jag2037 — CHANGES_REQUESTED** [review]
Changing the encoding outright is throwing an error for Schifflab txt files on my machine. Using a try/except clause instead should maintain compatibility across machines. This first tries to execute the first block of code (the "try" statement) and if a UnicodeDecodeError is thrown (this is the error with the edf files) it instead executes the second block of code (the "except" block).

File: sleepylyze/ioeeg.py (hunk 1 above)
Comment (nrecoder) [reply sleepylyze/ioeeg.py:144]: Great suggestion. This works on my machine with both the old Schiff Lab files and the shhs files text files.

**Comment by nrecoder** [comment]
> @nrecoder can you test the suggested code block on your machine before we merge the PR?

Great suggestion. This works on my machine with both the old Schiff Lab files and the shhs files text files.

**Comment by jag2037** [comment]
Great. Committing and merging now. Thanks!

**Pull request merged**